tool.html

```diff
@@ -48,20 +49,22 @@ <h3 class='open-sans-sans-serif-text'>Download MetaSUB data</h3>
 				</div>
 		 </form>
 		</div>
+		<hr>
 		<div class='align-text-center vertical-separator-from-top-50'>
 			<span>
 				<label for="fileInputAnalyze" class="button hover-effect font-size-20">Select a file</label>
 				<input type="file" id="fileInputAnalyze" file-picker/><br>
 			</span>
 			<h3 class='align-text-center open-sans-sans-serif-text'>Entropy Analysis</h3>
 		</div>
+		<hr>

```

Comment:
@MAZHARMIK - I think we should put this `<hr>` after the just below `<div>` tag, since while doing real-time analysis, the divider pulls out the dynamic chart outside the 'Entropy Analysis' section as seen in the below screenshot - 

![screenshot1](https://user-images.githubusercontent.com/14831291/35769915-aa781baa-0938-11e8-866f-d726c5e314b3.GIF)

Reply:
Ok sure. I will soon correct this. Thanks.